genkit-tools/common/src/runner/harness.ts

```diff
@@ -0,0 +1,14 @@
+import { glob } from 'glob';
+import * as path from 'path';
+import { logger } from '../utils';
+
+async function main() {
```

Comment:
Will this work with loading/transpiling TS files? Does it automatically set `GENKIT_ENV=dev`?

Reply:
Tested it with .ts, it worked. The envs are set automatically like in the default app loading.